src/vs/workbench/contrib/comments/browser/comments.contribution.ts

```diff
@@ -6,9 +6,20 @@
 import * as nls from 'vs/nls';
 import { InstantiationType, registerSingleton } from 'vs/platform/instantiation/common/extensions';
 import { Registry } from 'vs/platform/registry/common/platform';
-import 'vs/workbench/contrib/comments/browser/commentsEditorContribution';
 import { ICommentService, CommentService } from 'vs/workbench/contrib/comments/browser/commentService';
```

Comment:
I had to remove this because was getting an error about duplicate import statements. I think I've fixed that with this?

https://github.com/microsoft/vscode/pull/193420/commits/999e245c52594b33f915bac88b251fc477c1238c

Reply:
moved them to a different file